docs/static/arcsight-module.asciidoc

```diff
@@ -17,6 +17,22 @@ With a single command, the module taps directly into the ArcSight Smart Connecto
 parses and indexes the security events into Elasticsearch, and installs a suite of Kibana dashboards 
 to get you exploring your data immediately. 
 
+[[arcsight-prereqs]]
+==== Prerequisites
+
+These instructions assume that:
+
+* Logstash, Elasticsearch, and
+Kibana are already installed. The products you need are
+https://www.elastic.co/downloads[available to download] and easy to install. The
+Elastic Stack 5.6 or later is required for this module.  
+If you are using the Elastic Stack 6.2 and earlier, please see the instructions for those versions.
+
+* Logstash, Elasticsearch, and Kibana are running locally with default ports exposed, namely
```

Comment:
This is kinda confusing that we explicitly mention running locally under the pre-reqs section. I had to read carefully that this was under the header of "These instructions assume that: " . 

Maybe this should move "These instructions assume that:" down here inside of a note ?

Reply:
@jakelandis Trying again to address your concerns.